src/rust/generators/osquery-subgraph-generator/src/parsers/grapl_pack/files.rs

```diff
@@ -94,6 +91,18 @@ impl TryFrom<OSQueryResponse<OSQueryFileQuery>> for GraphDescription {
         graph.add_node(asset);
         graph.add_node(subject_file);
 
-        Ok(graph)
+        graph
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::parsers::OSQueryEvent;
+
+    #[test]
+    fn parse_pack_grapl_files_json() {
+        let test_json = r#"{"name":"pack_grapl_files","hostIdentifier":"ip-172-31-46-189.us-east-2.compute.internal","calendarTime":"Thu Oct 22 21:05:26 2020 UTC","unixTime":1603400726,"epoch":0,"counter":0,"numerics":false,"columns":{"action":"ACCESSED","atime":"1599180507","category":"temp","ctime":"1603395939","gid":"0","hashed":"0","inode":"101","md5":"","mode":"1777","mtime":"1603395939","sha1":"","sha256":"","size":"4096","target_path":"/tmp/","time":"1603395948","transaction_id":"0","uid":"0"},"action":"removed"}"#;
```

Comment:
minor nitpick: feels like something like this (many characters-long line) might be better off as a fixture file that gets read

Reply:
this might be what read_osquery_test_data is for? though that seems to take in zstd'd stuff

Reply:
I'm into that idea, putting this into a file. I'll do that, thanks.

Reply:
An easy way to do this is to put it in a file and then use the `include_str!` macro:
https://doc.rust-lang.org/std/macro.include_str.html

Reply:
@colin-grapl oh cool. i'll keep in the mind for next time, but i already committed a solution.